app/models/estimate_model.rb

```diff
@@ -4,6 +4,7 @@ class EstimateModel
   include SessionPersistable
 
   ESTIMATE_BOOLEANS = %i[upper_tribunal
```

Comment:
I should have removed this `upper_tribunal` boolean when I got rid of the tribunal screen. Sorry. To avoid further confusion, can you remove it from the list here?